### PR TITLE
feat: change diff scene route

### PR DESCRIPTION
**Pull request by josephaxisa**
from `/diff/:specKey/:compareSpecKey` to `/:specKey/diff/:compareSpecKey `in order for it to follow the same pattern as all other routes

### CODE REVIEW

File: packages/api-explorer/e2e/diffScene.spec.ts [hunk 1]
```diff
@@ -44,8 +44,9 @@ describe('Diff Scene', () => {
     await jestPuppeteer.resetBrowser()
     await page.setDefaultNavigationTimeout(120000)
   })
-  it('loads the default scene (/diff/3.1)', async () => {
-    await goToPage(`${BASE_URL}/diff/3.1`)
+
+  it('loads the default scene (/3.1/diff)', async () => {
```
Comment (jkaster) [inline packages/api-explorer/e2e/diffScene.spec.ts:48]: reminds me we need to change this default .. it should be 4.0 already based on spec status

File: packages/api-explorer/src/ApiExplorer.tsx [hunk 2]
```diff
@@ -149,7 +149,7 @@ export const ApiExplorer: FC<ApiExplorerProps> = ({
   const themeOverrides = adaptor.themeOverrides()
 
   let neededSpec = location.pathname?.split('/')[1]
-  if (!neededSpec || neededSpec === diffPath) {
+  if (!neededSpec) {
```
Comment (jkaster) [inline packages/api-explorer/src/ApiExplorer.tsx:152]: much better

**Review by jkaster — APPROVED** [review]
Nice cleanup. I like the consistent pattern, too.

File: packages/api-explorer/e2e/diffScene.spec.ts (hunk 1 above)
Comment (josephaxisa) [reply packages/api-explorer/e2e/diffScene.spec.ts:48]: agreed

**Pull request merged**